packages/examples/.storybook/main.ts

```diff
@@ -48,3 +37,7 @@ const config: StorybookConfig = {
   },
 };
 export default config;
+
+function getAbsolutePath(value: string): string {
+  return dirname(require.resolve(join(value, "package.json")));
```

Comment:
Given we're using absolute directory I'm assuming that means config paths support backslash on Windows?

Reply:
Yeah, I am actually having facade checked out on my windows machine :)